code/modules/mob/hear_say.dm

```diff
@@ -167,7 +167,22 @@
 
 /mob/proc/hear_sleep(var/message)
 	var/heard = ""
+	var/opentag = 0
+	var/closetag = 0
 	if(prob(15))
+		while(1) //Remove the HTML formatting so people don't hear colour
```

Comment:
I don't think it's sanitize_html. It's either strip_html_simple or strip_html_properly.